tests/baselines/reference/symbolLinkDeclarationEmitModuleNames.errors.txt

```diff
@@ -0,0 +1,28 @@
+tests/cases/compiler/monorepo/context/src/bindingkey.ts(1,29): error TS2307: Cannot find module '@loopback/context'.

```

Comment:
By the way, the changed baseline shows a bug in the test. It always worked because of the caching bug. Now it can no longer resolve the module. Should I fix that test?

Reply:
Yes please, Can you please change import to context instead? Thanks.

Reply:
I changed the path where the symlink is located so it can be resolved now. I didn't want to change any import paths as I don't know exactly what this test does.